sklearn/ensemble/_stacking.py

```diff
@@ -201,6 +201,13 @@ def fit(self, X, y, sample_weight=None):
         names, all_estimators = self._validate_estimators()
         self._validate_final_estimator()
 
+        # This is a quick fix to make StackingClassifier and StackingRegressor
+        # pass the tests despite of metadata routing for VotingClassifer and
```

Comment:
```suggestion
        # pass the tests despite of metadata routing for VotingClassifier and
```

Reply:
I think that we miss a verb. I assume that the reason here is that we don't support metadata routing for stacking while supporting for voting.